TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Shooter.java

```diff
@@ -0,0 +1,69 @@
+package org.firstinspires.ftc.teamcode;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+import com.qualcomm.robotcore.hardware.Gamepad;
+
+@TeleOp(name = "Shooter", group = "LinearOpMode")
+public class Shooter extends LinearOpMode {
+    @Override
+    public void runOpMode() throws InterruptedException {
+        DcMotor leftShooterMotor = hardwareMap.dcMotor.get("leftShooterMotor");
+        DcMotor rightShooterMotor = hardwareMap.dcMotor.get("rightShooterMotor");
+
+        rightShooterMotor.setDirection(DcMotorSimple.Direction.REVERSE);
+
+        waitForStart();
+
+        //Shooters start at 10% power
+        double leftShooterPower = .1;
```

Comment:
It may be worth creating java Constants called `STARTING_SHOOTER_POWER = 0.1`, so it's a bit more cleaner in code